arch/x86/core/Kconfig.ia32

```diff
@@ -170,4 +170,12 @@ config X86_USE_THREAD_LOCAL_STORAGE
 	help
 	  Internal config to enable thread local storage.
 
+config MFENCE_INSTRUCTION_SUPPORTED
```

Comment:
Shouldn't this have an `X86_` prefix like all the other Kconfig symbols here?

Reply:
MFENCE is x86-specific thing, X86_ prefix seems not necessary, I think.
in this Kconfig, there're still some symbols having no X86_ prefix for X86-specific IPs, like SSE/FPU.
In fact, this Kconfig has an old symbol called CLFLUSH_INSTRUCTION_SUPPORTED that is deleted now. I just follow that one.

Reply:
If you look at the description, `SSE` is deprecated and `X86_SSE` should be used instead. Having a `X86_` prefix makes it easier to distinguish for those who are not familiar with x86 instruction set. Makes it easier to grep too.

Reply:
OK. add X86 prefix.